sale_timesheet_order_line_no_update/models/project.py

```diff
@@ -33,15 +35,17 @@ def _compute_sale_line_id_domain(self):
                 field, value = "order_partner_id", rec.partner_id.id
             else:
                 field, value = "order_id", rec.sale_order_id.id
-            rec.sale_line_id_domain = [
-                ("is_service", "=", True),
-                ("is_expense", "=", False),
-                (field, "=", value),
-                ("state", "in", ["sale", "done"]),
-                "|",
-                ("company_id", "=", False),
-                ("company_id", "=", rec.company_id.id),
-            ]
+            rec.sale_line_id_domain = json.dumps(
```

Comment:
For this to work, the field should be binary, not char AFAIK.

Reply:
According to the docs it is Char: https://github.com/OCA/web/blob/14.0/web_domain_field/readme/USAGE.rst

Reply:
@pedrobaeza shall we go?

Reply:
This can be done without requiring such module with a binary field.